### PR TITLE
feat: Badges added

### PR DESCRIPTION
**Pull request by Manishsharma203**
For preview refer below link:
```https://codesandbox.io/s/badges-crux-3rxfo?file=/src/badges.css```

### CODE REVIEW

File: Components/index.html [hunk 1]
```diff
@@ -1,16 +1,70 @@
...
+          Button
+          <span class="badge badge-disable badge-round">4</span>
+        </button>
+        <br />
+        <br />
+        <button
+          style="
+            padding: var(--space-xs);
+            outline: none;
+            border: 0px;
```
Comment (Asuraking1n) [inline Components/index.html]: Instead of using border 0px you can use border:none

**Review by Asuraking1n — COMMENTED** [review]
You should give some top margin , they are too close I guess. Everything else looks good

File: Components/index.html (hunk 1 above)
Comment (Manishsharma203) [reply Components/index.html]: sure

**Review by jatinmore — APPROVED** [review]
looks good to me

**Pull request merged**